backend/apps/incidences/views.py

```
@@ -29,23 +29,17 @@ def get_serializer_class(self):
     
     def get_queryset(self):
         user = self.request.user
-        queryset = Incidence.objects.select_related('student', 'assigned_staff__user').all()
+        queryset = Incidence.objects.filter(is_active=True).select_related('student', 'assigned_staff__user')
 
-        if user.is_superuser:
-            return queryset
+        if user.is_staff: return queryset
+        
+        roles = [r.lower() for r in user.memberships.filter(is_active=True).values_list('role__name', flat=True)]
+        if "admin" in roles: return queryset
```

Ruff (0.15.6)

[error] Multiple statements on one line (colon)

(E701)